apps/to-do-api/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "to-do-api",
+  "version": "0.0.0",
+  "description": "Azure Function REST API for To Do List",
+  "license": "MIT",
+  "main": "./dist/main.js",
```

we should use ESM by default (`"type": "module"`)

```suggestion
  "main": "./dist/main.js",
  "type": "module",
```